diff --git a/dist/metaid.js b/dist/metaid.js
index 48f0e1f87f1c465707b53ac6fa3880b10e356b06..9e61c16ee30acd352c8e01d3e1685e9492036970 100644
--- a/dist/metaid.js
+++ b/dist/metaid.js
@@ -33888,7 +33888,8 @@ let MvcConnector$1 = class MvcConnector2 {
       }
     }
     return {
-      txid: payRes[payRes.length - 1].getTxId()
+      txid: payRes[payRes.length - 1].getTxId(),
+      allTxid:payRes.map(item=>item.getTxId())
     };
   }
   async updateUserInfo({
